lib/util.js

```diff
@@ -159,9 +159,9 @@ function stylizeNoColor(str, styleType) {
 
 
 function arrayToHash(array) {
-  var hash = {};
+  var hash = Object.create(null);;
 
-  array.forEach(function(val, idx) {
+  array.forEach(function(val) {
```

Comment:
Ok cool. Yeah I guess I agree, the naming of `Array.prototype.reduce` is definitely a bit awkward here. Maybe the general arrow function stance is also worth a larger style/convention conversation elsewhere? Thanks for the input!